### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abdounikarim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ LexikJWTAuthenticationBundle
 
 This bundle provides JWT (Json Web Token) authentication for your Symfony API.
 
-It is compatible and tested with PHP 5.5, 7.2, 7.3, on Symfony 3.4 and 4.2.
+It is compatible and tested with PHP 5.5, 7.2, 7.3, on Symfony 3.4, 4 and 5.
```
Comment (elangovanshanthi) [inline README.md]: @abdounikarim Did you verified, it will work for Symfony 4 and symfony5. I have tried, it is not worked for symfony4.3.

Comment (Spomky) [reply README.md]: Yes its verified by tests: https://travis-ci.org/lexik/LexikJWTAuthenticationBundle

**Comment by t-richard** [comment]
@abdounikarim @elangosundar @Spomky could this be merged ? Had people on the Symfony Slack thinking the bundle was not maintained anymore because the README didn't stated that it was compatible :slightly_smiling_face: 
Maybe we could rephrase to point to the travis for more details
Something like 
> It is compatible and tested with PHP 5.5, 7.2, 7.3, on Symfony 3.4, 4 and 5. To see specific versions this bundle is tested against, please refer to our [Travis builds](https://travis-ci.org/lexik/LexikJWTAuthenticationBundle).

**Comment by abdounikarim** [comment]
@t-richard i don’t have rights to merge.

**Comment by chalasr** [comment]
Thank you @abdounikarim.

**Pull request merged**